samples/esb/esb_prx/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _esb_prx:
+
+Enhanced ShockBurst: Receiver
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows the Enhanced ShockBurst Receiver application.
```

Comment:
Rather than restating the sample's name, maybe briefly explaining what an ESB Receiver is would give some context to a user who bumps on this page from search?